### PR TITLE
Bug/local_quali_campaign

### PR DESCRIPTION
**Pull request by djmbritt**
There was an issue getting the inclusive props and exlusive props to load.
This was happening when there was a hard refresh of the page.
Now that these props have been moved to the computed property it is behaving as expected.

### CODE REVIEW

**Review by laurensV — CHANGES_REQUESTED** [review]
@djmbritt Did you test the `canUserQualify`? It seems like you removed the check to fill in canUserQualify by removing the getQualifications function, now canUserQualify is always false it seems, but its still being used in the template

**Comment by djmbritt** [comment]
Fixed for batches and campaigns

File: pages/campaigns/_id/_batch/index.vue [hunk 1]
```diff
@@ -727,19 +778,6 @@ export default {
         }
       }
       this.canUserQualify = this.checkUserQualify()
```
Comment (laurensV) [inline pages/campaigns/_id/_batch/index.vue:780]: this line still calls the function checkUserQualify which doesn't exists anymore and overwrites a computed property. will definitaly cause an error, make sure to test these kind of things

**Review by laurensV — CHANGES_REQUESTED** [review]

File: pages/campaigns/_id/_batch/index.vue (hunk 1 above)
Comment (djmbritt) [reply pages/campaigns/_id/_batch/index.vue]: Fixed ✅

**Pull request merged**